spec/jekyll-feed_spec.rb

```diff
@@ -43,44 +43,44 @@
   end
 
   it "puts all the posts in the feed.xml file" do
-    expect(contents).to match /http:\/\/example\.org\/updates\/2014\/03\/04\/march-the-fourth\.html/
-    expect(contents).to match /http:\/\/example\.org\/news\/2014\/03\/02\/march-the-second\.html/
-    expect(contents).to match /http:\/\/example\.org\/news\/2013\/12\/12\/dec-the-second\.html/
+    expect(contents).to match %r!http://example.org/updates/2014/03/04/march-the-fourth.html!
```

Comment:
As tests are passing, it doesn't look like it's needed.

Reply:
Agreed. But it would still be an *incorrect regex* technically:
```bash
irb>
  regex = %r!http://example.org/!
  regex === "http://example.org/"
  # => true
  regex === "http://examplesorg/"
  # => true
```

Reply:
Yeah, let's escape the periods! They need to be literal periods, but in regexp a period means "any character".